index.html

```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
+            <li class="tag-btn"> <button type="button"  class="btn-small">css</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button"  class="btnNormal">See Project</button>
+            
+        </div>
+
+        <!-- card ends here -->
+
+        <!-- single card start here -->
+        
+        <div class="card third-card"> 
+            <img src="cardThreeImage.svg" alt="" class="card-img">
+
+        <h2 class="project-title">Tonic</h2>
+
+        <ul class="workInfo">
+            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">Back End Dev</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">2015</a></li>
+        </ul>
+
+        <p class="details-sec">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="tag-btn-container">
+            <li class="tag-btn"> <button type="button"  class="btn-small">html</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">css</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button" class="btnNormal">See Project</button>
+            
+        </div>
+
+        <!-- card ends here -->
+
+        <!-- single card start here -->
+        
+        <div class="card fouth-card"> 
+            <img src="cardFourImage.svg" alt="" class="card-img">
+
+        <h2 class="project-title">Multi-Post
+            Stories</h2>
+
+        <ul class="workInfo">
+            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">Back End Dev</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">2015</a></li>
+        </ul>
+
+        <p class="details-sec">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="tag-btn-container">
+            <li class="tag-btn"> <button type="button"  class="btn-small">html</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">css</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button"  class="btnNormal">See Project</button>
```

Comment:
- [ ] Please fix margins (top, bottom, left, right) defined in the design

| Your design | Figma design |
| ------------- | --------------- |
| ![image](https://user-images.githubusercontent.com/74744787/164705285-603ae00a-875e-4f62-9e70-96152e779d65.png) | ![image](https://user-images.githubusercontent.com/74744787/164705377-767bfc11-f62f-42e8-b87c-bd9dc8a0f312.png) |